livekit-plugins/livekit-plugins-anthropic/livekit/plugins/anthropic/llm.py

```diff
@@ -560,24 +562,26 @@ def type2str(t: type) -> str:
             raise ValueError(f"unsupported type {t} for ai_property")
 
         p: dict[str, Any] = {}
-        if arg_info.default is inspect.Parameter.empty:
+        if not _is_optional_type(arg_info.type)[0]:
```

Comment:
I think the logic is different?
Does it still accept default values?  `my_var: int = 5 `

Reply:
Can we add a test for it as well? (for default values)

Reply:
we should count it as optional right?

Reply:
Yes, when sending it to OAI, they should not be inside the "`required`" field. Tho the type itself inside the Python code isn't really "optional"